### PR TITLE
Case 21029: Fix duplicating non-dynamic grabbed entities in edit.js

### PR DESCRIPTION
**Pull request by huffman**
https://highfidelity.fogbugz.com/f/cases/21029/Near-grabbed-non-dynamic-colliding-entity-that-is-duplicated-with-Create-will-put-its-current-linear-velocity-on-the-duplicate

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/17218/HighFidelity-Beta-PR14902-2728654.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/17218/HighFidelity-Beta-PR14902-2728654.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/17218/HighFidelity-Beta-Interface-PR14902-2728654.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/17218/HighFidelity-Beta-PR14902-2728654.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/17218/HighFidelity-Beta-Interface-PR14902-2728654.exe

File: scripts/system/libraries/entitySelectionTool.js [hunk 1]
```diff
@@ -284,6 +284,8 @@ SelectionManager = (function() {
                     properties.parentJointIndex = null;
                     properties.localPosition = properties.position;
                     properties.localRotation = properties.rotation;
+                    properties.velocity = { x: 0, y: 0, z: 0 };
```
Comment (dback2) [inline scripts/system/libraries/entitySelectionTool.js]: Small nit-picky but maybe Vec3.ZERO instead?

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/17283/HighFidelity-Beta-PR14902-87b9dff.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/17283/HighFidelity-Beta-PR14902-87b9dff.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/17283/HighFidelity-Beta-Interface-PR14902-87b9dff.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/17283/HighFidelity-Beta-PR14902-87b9dff.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/17283/HighFidelity-Beta-Interface-PR14902-87b9dff.exe

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/17292/HighFidelity-Beta-PR14902-e800a6e.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/17292/HighFidelity-Beta-PR14902-e800a6e.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/17292/HighFidelity-Beta-Interface-PR14902-e800a6e.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/17292/HighFidelity-Beta-PR14902-e800a6e.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/17292/HighFidelity-Beta-Interface-PR14902-e800a6e.exe

**Pull request merged**